src/Akka.HealthCheck.Persistence/AkkaPersistenceLivenessProbe.cs

```diff
@@ -81,25 +81,9 @@ private CheckTimeout()
         }
     }
 
-    internal sealed class WarmupComplete
-    {
-        public static readonly WarmupComplete Instance = new();
-        private WarmupComplete(){ }
-    }
-
-    internal sealed class WarmupFailed
-    {
-        public WarmupFailed(Exception cause)
-        {
-            Cause = cause;
-        }
-
-        public Exception Cause { get; }
-    }
-
     public class AkkaPersistenceLivenessProbe : ActorBase, IWithTimers
     {
-        public const string PersistenceId = "Akka.HealthCheck.Probe";
+        public static readonly string PersistenceId = $"Akka.HealthCheck-{Guid.NewGuid()}";
```

Comment:
This makes sure that each probe has a unique persistence id.

Reply:
LGTM